content/02.body.md

```diff
@@ -1,13 +1,44 @@
+The consequence of degree bias is that a difference in degree between two nodes may not reflect a difference in the number of true edges.
+Reliance on degree may be unfavorable depending on the prediction/analysis task being conducted and the magnitude of possible degree bias in the data.
+
+Another reason why a method's reliance on degree can be unfavorable is that degree imbalance can lead to prediction nonspecificity.
+Nonspecific predictions are made on the basis of generic characteristics rather than the specific connectivity information contained in a network.
+For example, Gillis et al. [@doi:10.1371/journal.pone.0017258] examined the concept of prediction specificity in the context of gene function prediction and found that many predictions appear to rely primarily on multifunctionality and could be "potentially misleading with respect to causality."
+Real networks have a variety of degree distributions (Figure {@fig:hetionet}), and they commonly exhibit degree imbalance [@doi:10.1371/journal.pone.0017645; @doi:10.1007/978-1-61779-361-5_13; @doi:10.1038/s41467-019-08746-5].
+Degree imbalance leads high-degree nodes to dominate in the predictions made by degree-associated methods [@doi:10.1093/bioinformatics/btv215], which are effective predictors of connections in some biological networks [@doi:10.1186/1752-0509-2-11].
+
+![Degree distribution can vary greatly between networks, even between the source and target degree distributions within the same (directed or bipartite) network.
+  Shown are seven degree distributions for six edge type subnetworks of Hetionet [@doi:10.7554/eLife.26726].
+  (The final two are the source and target distributions of the gene-regulates-gene directed edge type.)
+](https://github.com/greenelab/xswap-analysis/raw/5a257c9d28e0a4c3cc8f2fd8493f27ff4652f2c6/img/hetionet_degrees.png){#fig:hetionet width="100%"}
+
+Consequently, degree-based predictions are more likely nonspecific, meaning the same set of predictions performs well for different tasks.
+However, depending on the prediction task, edge predictions between very high degree nodes may be undesired, uninsightful, or nonspecific.
+Model evaluation is challenging in this context, as nonspecific or trivial predictions need not be incorrect, even if they are not the desired outputs of the predictive model.
+For example, predicting that the highest degree node in a network shares edges with the remaining nodes to which it is not connected will often lead to many correct predictions, despite the predictions being generic to all nodes in the network.
+To clarify the situation, degree-based features must be included in the interpretation of predictions in order to disentangle desired from non-desired effects and to more effectively evaluate and compare predictive models.
+
+Degree is an important metric for differentiating between nodes.
```

Comment:
I think that you could remove this and the three following sentences. Perhaps they fit somewhere else in the intro there.